src/rules/noInvalidTemplateStringsRule.ts

```diff
@@ -50,10 +50,19 @@ function walk(ctx: Lint.WalkContext<void>) {
     });
 
     function check(node: ts.StringLiteral): void {
-        const idx = node.text.search(/\$\{/);
-        if (idx !== -1) {
+        const index = node.text.search(/\$\{/);
+        if (index !== -1) {
+            /**
+             * Support for ignoring case: '\${template-expression}'
+             */
+            const unescapedText: string = node.getFullText();
+            const preceedingCharacter = unescapedText.substr(unescapedText.search(/\$\{/) - 1, 1);
+            if (preceedingCharacter === "\\") {
```

Comment:
Because the characters preceeding the lines' ${} expressions are backslashes. "\\" is an escaped backslash. Could be made clearer somehow probably.